frontend/inbox/src/pages/Inbox/QuickFilter/Popup.tsx

```diff
@@ -18,13 +18,15 @@ import {useTranslation} from 'react-i18next';
 
 function mapStateToProps(state: StateModel) {
   const channels: Channel[] = Object.values(allChannels(state));
+  const conversations = Object.values(state.data.conversations.all.items);
+
   return {
     user: state.data.user,
     filter: state.data.conversations.filtered.currentFilter,
     tags: state.data.tags.all,
     channels,
-    sources: channels.reduce<Set<string>>((acc, {source}) => {
-      acc.add(source);
+    sources: conversations.reduce<Set<string>>((acc, {channel}) => {
+      acc.add(channel.source);
```

Comment:
this fixes the problem of not seeing sources which channels are disconnected in the filtering box